pkg/kubelet/logs/container_log_manager.go

```diff
@@ -171,20 +173,28 @@ func NewContainerLogManager(runtimeService internalapi.RuntimeService, osInterfa
 			MaxSize:  parsedMaxSize,
 			MaxFiles: maxFiles,
 		},
-		clock: clock.RealClock{},
-		mutex: sync.Mutex{},
+		clock:            clock.RealClock{},
+		mutex:            sync.Mutex{},
+		maxWorkers:       maxWorkers,
+		queue:            workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "kubelet_log_rotate_manager"),
+		monitoringPeriod: monitorInterval,
 	}, nil
 }
 
 // Start the container log manager.
 func (c *containerLogManager) Start() {
 	ctx := context.Background()
+	klog.InfoS("Initializing container log rotate workers", "workers", c.maxWorkers, "monitorPeriod", c.monitoringPeriod)
+	for i := 0; i <= c.maxWorkers; i++ {
```

Comment:
it is more natural and consistent with other worker counts to make maxWorkers be the number of workers and make this:
```suggestion
	for i := 0; i < c.maxWorkers; i++ {
```

Reply:
Ack